src/query/sql/src/planner/binder/ddl/table.rs

```diff
@@ -468,6 +469,19 @@ impl Binder {
             options.insert("TRANSIENT".to_owned(), "T".to_owned());
         }
 
+        // todo(geometry): remove this when geometry stable.
+        if let Some(CreateTableSource::Columns(cols)) = &source {
+            if cols
+                .iter()
+                .any(|col| matches!(col.data_type, TypeName::Geometry))
+                && !self.ctx.get_settings().get_enable_geo_create_table()?
+            {
+                return Err(ErrorCode::GeometryError(
+                    "Create table using the geometry type is an experimental feature. We do not guarantee its compatibility.",
```

Comment:
error message should add `set enable_geo_create_table=1` to notice user modify setting.